packages/scheduler/src/components/FullCalendar/FullCalendar.js

```diff
@@ -146,13 +147,16 @@ const FullCalendar = ({
     })
   }
 
+  const getFormattedTime = date =>
```

Comment:
some documentation about expected input and output would be useful.
maybe introduce more global util, if it could be used somewhere else as well?